docs/trash.md

```diff
@@ -0,0 +1,97 @@
+Trash
+=====
+
+Under normal circumstances, when an item in Box is deleted, it is not actually erased immediately. Instead, it is
+moved to the Trash. The Trash allows you to recover files and folders that have been deleted. By default, items in
+the Trash will be purged after 30 days.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [List Trashed Items](#list-trashed-items)
+- [Get Trashed File, Folder or WebLink](#get-trashed-file-folder-or-weblink)
+- [Restore File, Folder, or WebLink from Trash](#restore-file-folder-or-weblink-from-trash)
+- [Permanently Delete File, Folder or WebLink](#permanently-delete-file-folder-or-weblink)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+List Trashed Items
+------------------
+
+To retrieve a list of all trashed items, you can use `client.trash().get_trashed_items(offset=None, limit=None, fields=None)`
+
+```python
+trashed_items = client.trash().get_trashed_items()
+for trashed_item in trashed_items:
+    # Do something
+```
+
+Get Trashed File, Folder or WebLink
```

Comment:
I think it's okay to just say "Trashed Item" here and in the other section headings — it should be well-understood what types are Items